roles/kafka_broker/tasks/rbac.yml

```diff
@@ -21,12 +21,12 @@
 # Take super.users property, split into list, remove empty string from list, and add in mds_super_user
 - name: Initialize Super Users List
   set_fact:
-    super_users: "{{ (kafka_broker_final_properties['super.users'] | default('')).split(';') | difference(['']) }} + [ 'User:{{ mds_super_user }}' ]"
+    super_users: "{{ (kafka_broker_final_properties['super.users'] | default('')).split(';') | difference(['']) + [ 'User:' + lookup('vars', 'mds_super_user', default=mds_super_user) ]}}"
```

Comment:
Thanks! LGTM
Small query: 
If the lookup fails (unlikely) and the default value is picked, would that be the string 'mds_super_user'?

Reply:
good point, 
`default=mds_super_user` would actually lookup for value of mds_super_user variable itself. 
discussed with team, removing the default in above lookup (as it already has default value mentioned in VARIABLES.md which is 'mds')